razzle.config.js

```diff
@@ -8,15 +11,21 @@ module.exports = {
         A prod build will fail if the API changes so it is fairly safe.
       */
       appConfig.module.rules.shift();
-      // This is to pass the bundlde performance test in CI
-      appConfig.performance = Object.assign(
-        {},
-        {
-          maxAssetSize: 350000,
-          maxEntrypointSize: 350000,
-        },
+    } else if (target === 'node' && dev) {
```

Comment:
This currently will be running on the server and not on the client.  To run for client-side code, the `target` will need to be `web`.